mmv1/third_party/terraform/tests/resource_alloydb_backup_test.go

```diff
@@ -157,3 +157,78 @@ resource "google_service_networking_connection" "vpc_connection" {
 }
 `, context)
 }
+
+// Test to create on-demand backup with mandatory fields.
+func TestAccAlloydbBackup_createBackupWithMandatoryFields(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": RandString(t, 10),
+	}
+
+	VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckAlloydbBackupDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAlloydbBackup_createBackupWithMandatoryFields(context),
+			},
+		},
+	})
+}
+
+func testAccAlloydbBackup_createBackupWithMandatoryFields(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_alloydb_backup" "default" {
+  backup_id    = "tf-test-alloydb-backup%{random_suffix}"
+  location = "us-central1"
+  cluster_name = google_alloydb_cluster.default.name
+  depends_on = [google_alloydb_instance.default]
+}
+
+resource "google_alloydb_cluster" "default" {
+  location = "us-central1"
+  cluster_id = "tf-test-alloydb-cluster%{random_suffix}"
+  network    = "projects/${data.google_project.project.number}/global/networks/${google_compute_network.default.name}"
```

Comment:
These can be replaced with `network = google_compute_network.default.id` I believe. Does that work?

Reply:
done